crowbar_framework/app/controllers/installer/upgrades_controller.rb

```diff
@@ -323,9 +323,14 @@ def post_cleanup_repos
           false
         )
 
-        Crowbar::Repository.chef_data_bag_destroy(
-          "#{repository.data_bag_name}/#{repository.id}"
-        )
+        begin
+          Crowbar::Repository.chef_data_bag_destroy(
+            "#{repository.data_bag_name}/#{repository.id}"
+          )
+        rescue Net::HTTPServerException
+          # it is fine to pass if the databag doesn't exist
```

Comment:
yes it really works. we are doing it like that in multiple places where we make chef rest calls